### ISSUE TITLE
http://a9.com/-/spec/opensearch/1.1/ not found

### ISSUE THREAD

**Issue by braco** [dewitt/opensearch#3]
Not sure where to ask this, but any idea what happened to 

http://a9.com/-/spec/opensearch/1.1/

?

It's 404 now.

**Comment by dewitt** [comment]
I don't believe that URL has resolved for years. While not great, it's something we're unlikely to be able to fix since we don't control the domain. Fortunately it's just a name (in the form of a URI) and not required to return anything when dereferenced to be valid, if I'm reading [this](https://www.w3.org/TR/xml-names) and [this](http://www.rfc-editor.org/rfc/rfc3986.txt) correctly.

Please see [here](
https://groups.google.com/d/msg/opensearch/8ecuX35-lEs/JR_cs0N6k68J
) for the brief previous discussion.

**Comment by vladak** [comment]
For the record, the response for the URL is now HTTP error 403.

**Comment by svenha** [comment]
And now it redirects to https://www.amazon.com/-/spec/opensearch/1.1/ , which is really annoying.

**Comment by svenha** [comment]
Today, I found out that the above redirect leads to 404 page.
